.github/scripts/script.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+set -xmveuo pipefail
+
+group=libvirt
+# Re-launch the script with libvirt as the primary group.
+# Needed because we just created the group and added the GHA runner use to it.
+# Avoids running vagrant as root.
+if [ $(id -gn) != $group ]; then
+  exec sg $group "$0 $*"
+fi
+
+sed -i -e 's/memory: 10500/memory: 5500/g' vagrant/boxes.d/*
+sed -i -e 's/cpus: 4/cpus: 2/g' vagrant/boxes.d/*
```

Comment:
does GHA not handle 4?

Reply:
GHA only has 2 virtual CPUs, just like Travis.

These 2 lines only have any effect if when we run `pulp2-nightly-pulp3-source-centos7` on CI, which we currently do not do.